Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoEmptyBlockRule.swift

```diff
@@ -54,6 +55,42 @@ struct NoEmptyBlockRule: Rule {
             Example("""
             while i < 10 { /* do something */ }
             """),
+            Example("""
+            var flag = true {
+                willSet {}
+            }
+            """, configuration: ["disabled_block_types": ["function_bodies"]]),
+            Example("""
+            func f() {}
+            """, configuration: ["disabled_block_types": ["function_bodies"]]),
+            Example("""
+            deinit {}
+            """, configuration: ["disabled_block_types": ["initializer_bodies"]]),
+            Example("""
+            init() {}
+            """, configuration: ["disabled_block_types": ["initializer_bodies"]]),
+            Example("""
+            for _ in 0..<10 {}
+            """, configuration: ["disabled_block_types": ["statement_blocks"]]),
+            Example("""
+            do {
+            } catch {
+            }
+            """, configuration: ["disabled_block_types": ["statement_blocks"]]),
+            Example("""
+            defer {}
+            """, configuration: ["disabled_block_types": ["statement_blocks"]]),
+            Example("""
+            if flag {
+            } else {
+            }
+            """, configuration: ["disabled_block_types": ["statement_blocks"]]),
+            Example("""
+            repeat {} while (flag)
+            """, configuration: ["disabled_block_types": ["statement_blocks"]]),
+            Example("""
+            while i < 10 {}
+            """, configuration: ["disabled_block_types": ["statement_blocks"]]),
```

Comment:
Let's merge all the examples for one specific type of block into one, so that the rendering is a bit more dense.